### ISSUE TITLE
BUG: `nys-radiobutton` missing visual focus ring

### ISSUE THREAD

**Issue by NovaCat35** [ITS-HCD/nysds#1293]
# Summary
Focus moves correctly on submit, but the visual focus ring does not appear on `nys-radiobutton`
